### ISSUE TITLE
Spacing is odd in the timeline

### ISSUE THREAD

**Issue by kittykat** [vector-im/element-android#3964]
Spacing between subsequent messages from one user looks too big (looks less odd when messages alternate between senders):

![Screenshot_20210903-231252.png](https://user-images.githubusercontent.com/51663/132070393-dcc9a025-6ce9-4018-9353-9e26fe7d512b.png)

### Acceptance criteria
- [ ] Spacing is reduced to be more compact, especially when multiple messages have a similar timestamp

**Comment by bahur142** [comment]
On the old Riot.im it was good. Since Element the time stamp is on the next line, instead to be in the same line with the message. So I decided to switch off the option "Show timestamp for all messages".

**Comment by bmarty** [comment]
Yes, this is due to option "Show timestamp for all messages" enabled, and this is not a regression

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
@bmarty and @kittykat, if the time were upon the same line as the text, the view would be more compact, but because that would cause problems with long messages, I suggest that it merely be oriented to the left (as all text should be; the locale should decide that) so that eye-strain is reduced, and the list appears more compact despite not actually being more compact. Its current implementation causes ascertainment of which message the timestamp refers to to be difficult.
